### ISSUE TITLE
Probe to bone test positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10656]
**Preferred term label:**
Probe to bone test positive

**Synonyms**
Probe inserted into ulcer touches bone

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Related to https://hpo.jax.org/browse/term/HP:0002754 and https://hpo.jax.org/browse/term/HP:0002754

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hand or foot osteomyelitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001105
- label: Positive probe-to-bone test
- parents: [Cutaneous wound (HP:0032674)](https://hpo.jax.org/app/browse/term/HP:0032674)
- pmid: PMID:27369321
- def: An abnormal result of the probe-to-bone (PTB) test. The PTB test is performed by inserting a sterile, metal surgical probe into the ulcer and is positive if a hard, gritty surface is felt inside. For large wounds, this may require gentle exploration of the wound with the probe.
- comment: The basis behind the PTB test is that if a probe can reach bone, so can bacteria.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10656
- synonyms: Abnormal probe-to-bone test(NONE)
